lib/rototiller/rake/dsl/dsl_extention.rb

```diff
@@ -1,14 +1,11 @@
-require 'rototiller/rake/tasks/rototiller_task'
+require 'rototiller/task/rototiller_task'
 
 module Rake
   module DSL
 
     def acceptance_task(*args, &block)
-      Rake::RototillerTask.define_task :acceptance, &block
-      description = "Tests in the 'Acceptance' tier"
-      unless Rake::RototillerTask[:acceptance].comment
-        Rake::RototillerTask[:acceptance].add_description(description)
-      end
+      desc "Tests in the 'Acceptance' tier" unless ::Rake.application.last_comment
```

Comment:
i don't understand this conditional.  doesn't this skip if there is *any* comment in the Rakefile?